scripts/startup_announce.js

```diff
@@ -0,0 +1,13 @@
+// Description:
+//   Announce when the bot starts up
+
+const target = process.env.HUBOT_STATUS_CHANNEL || 'bots';
+
+module.exports = (robot) => {
+  setTimeout(() => {
+    robot.messageRoom(target, {
+      target,
+      text: 'Johnny 5 is alliiiiiive!  Also I have just booted up.'
+    })
+  }, 2000);
```

Comment:
Why the timeout?

Reply:
The mistaken assumption that the module might be initialized before the Slack connection was ready and trying to send a message might create an error.  I removed the timeout and it continued working just fine, so...  updated commits below.  😄